lib/Marlin/Marlin/src/module/motion.cpp

```diff
@@ -1153,6 +1153,9 @@ feedRate_t get_homing_bump_feedrate(const AxisEnum axis) {
           #endif
 
           stealth_states.x = tmc_enable_stallguard(stepperX);
+          #if ENABLED(CRASH_RECOVERY)
+            stepperX.stall_sensitivity(crash_s.home_sensitivity[0]);
+          #endif
```

Comment:
This code is not supposed to be there. I recently replaced it with `crash_s.start_sensorless_homing_per_axis(axis);`
This also applies for Y axis.

I forgot to remove 

```
          #if ENABLED(CRASH_RECOVERY)
            stepperY.stall_sensitivity(crash_s.home_sensitivity[1]);
          #endif
```

Reply:
Do you want me to make a PR for this?